packages/editor/src/___dev-demo/standalone.tsx

```diff
@@ -1,4 +1,4 @@
-import { SerloEditor } from '@editor/package'
+import { SerloEditor } from '@editor/package/editor'
```

Comment:
Super minor:
I'm not sure about this one. The demo should simulate how another integration would use the editor so I'd expect an import from `@editor/package` with an ignore comment.